etcdctl/ctlv3/command/printer_table.go

```diff
@@ -49,5 +51,6 @@ func (tp *tablePrinter) DBStatus(r dbstatus) {
 	for _, row := range rows {
 		table.Append(row)
 	}
+	table.SetAlignment(tablewriter.ALIGN_CENTER)
```

Comment:
can these be right-aligned instead? usually numbers are right-aligned so it's easier to compare:
```
  123
12345
```
vs
```
 123
12345
```